### ISSUE TITLE
Total Intracranial volume estimates

### ISSUE THREAD

**Issue by akhanf** [khanlab/hippunfold#182]
TIV is a necessity for volumetry so would be nice to include an estimate in the output volume TSV files.

Could use the freesurfer approach which extracts it from a linear transformation to a template space.

**Comment by romeopenheiro** [comment]
Is it possible to apply the estimated Total Intracranial Volume, aka ICV, to the `anat` output of Hippunfold? I do not have the dicoms for the dataset (I only have the structural MRI : T1-MPRAGE) that are generally required to implement this. I am hoping to correct for the variance in brain volume among participants.

**Comment by akhanf** [comment]
Yes it will be fine to just apply the eTIV (e.g. from freesurfer) to anything in the native or cropped-native space (e.g. `space-T1w` or `space-cropT1w`) in the anat folder. 

The `space-corobl` images would already have an affine correction applied to them, unless the `--rigid_reg_template` flag was used, so for those, would not want to then apply eTIV correction since might over-correct.
